src/melee/ft/fighter.h

```diff
@@ -247,30 +271,39 @@ typedef struct _Fighter {
     u8 xE;
     u8 xF;
     s32 x10;
-    u8 data_filler_x10[0x2C - 0x14];
+    s32 x14;
+    s32 x18;
+    s32* x1C;
+    s32 x20;
+    s32 x24;
+    s32 x28;
     /* 0x2C */ f32 x2C_facing_direction;
     /* 0x30 */ f32 x30_facingDirectionRepeated;
     /* 0x34 */ Vec3 x34_scale;
-    u8 filler_x40[0x74 - 0x40];
-    /* 0x74 */ Vec3 x74_animVel;
-    /* 0x80 */ Vec3 x80_selfVel;
-    /* 0x8C */ Vec3 x8C_knockbackVel;
-    /* 0x98 */ Vec3 x98_AtkShieldKnockback;
-    /* 0xA4 */ Vec3 xA4_unkVel;
-    /* 0xB0 */ Vec3 xB0_pos;
-    /* 0xBC */ Vec3 xBC_prevPos;
-    /* 0xC8 */ Vec3 xC8_pos_delta;
-    /* 0xD4 */ Vec3 xD4_unkVel;
-    /* 0xE0 */ s32 xE0_ground_or_air;
-    /* 0xE4 */ f32 xE4_groundAccel1;
-    /* 0xE8 */ f32 xE8_groundAccel2;
-    /* 0xEC */ f32 xEC_groundVel;
-    /* 0xF0 */ f32 xF0_knockbackGroundVel;
-    /* 0xF4 */ f32 xF4_groundAttackerShieldKnockbackVel;
-    /* 0xF8 */ Vec2 xF8_playerNudgeVel;
-    /* 0x100 */ f32 x100;
+    f32 x40;
+    u8 filler_x40[0x74 - 0x44];
+    struct phys                                                // 0x74
+    {                                                          //
+        Vec3 x74_anim_vel;                                         // 0x74
+        Vec3 x80_self_vel;                                         // 0x80
+        Vec3 x8c_kb_vel;                                           // 0x8C
+        Vec3 x98_atk_shield_kb;                                // 0x98
+        Vec3 xA4_unk_vel;                                      // 0xA4
+        Vec3 xB0_pos;                                          // 0xb0
+        Vec3 xBC_prevPos;                                     // 0xBC
+        Vec3 pos_delta;                                        // 0xC8
+        Vec3 xD4_unk_vel;                                        // 0xD4
+        int xE0_airborne;                                         // 0xE0
```

Comment:
What's the motivation for the inner phys struct? Based on this assertion, there shouldn't be an inner struct here
https://github.com/doldecomp/melee/blob/a7d73625fe889bf925b03981602f9b184af23fef/src/melee/ft/ftlib.c#L973

Reply:
It just seemed to organize things a bit, and I saw the struct in one of Kalua's matches for fighter.  but based on this assertion you are right.  looks like no inner struct, and that looks to be the correct var name.